### PR TITLE
Transform to ES5

### LINKED ISSUE #177: Pre-compiled ES5 Version
I'm trying to use Indicative with create-react-app but getting minify errors.

After searching, it looks like you've added this via rollup. But I'm not sure how to use to install the version I need.

Is there a way to get the pre-complied version version requested here: https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify

Thanks!

### PR DESCRIPTION
**Pull request by mhemrg**
Closes #177 
Related to https://github.com/Slynova-Org/charmap/pull/4 and https://github.com/poppinss/haye/pull/3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,7 +54,7 @@
     "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-uglify": "^2.0.1",
     "standard": "^10.0.3",
-    "striptags": "^3.1.1",
+    "striptags": "^2.2.1",
```
Comment (thetutlage) [inline package.json:57]: May I know why we decreased the version of `striptags`?

**Review by thetutlage — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mhemrg) [reply package.json:57]: According to its docs, `v3+` targets ES6 and is incompatible with `uglifyjs`.

Comment (thetutlage) [reply package.json:57]: Yeah will be better, if you can add a proper reasoning behind these random changes.

**Comment by thetutlage** [comment]
Can you share some steps on how do I verify that it is compatible with ES5?

**Comment by mhemrg** [comment]
In order to transpile indicative to ES5, we have to have transpiled dependencies. I've decreased the version of `striptags` to 2 because it targets ES5. I've upgraded `@slynova/slug` and `haye` too because they're compatible with ES5 in they're latest versions.

We're using `uglify-es`, but if we use `uglifyjs` it will break. `uglifyjs` requires ES5 compatibility.

**Comment by 7ammer** [comment]
This saved my bacon. Thanks!

**Pull request closed without merging**